specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,959 @@
+        "operationId": "Features_Create",
+        "x-ms-examples": {
+          "PostFeature": {
+            "$ref": "./examples/features/Features_Create.json"
+          }
+        },
+        "consumes": [
+          "application/geo+json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/FeatureBody"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/CreateFeatureResult"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/features/datasets/{datasetId}/collections/{collectionId}/items/{featureId}": {
+      "get": {
+        "description": "The Get Feature API returns the feature identified by the provided id in the given collection.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/create). Datasets consist of multiple feature collections, which are a collection of features of a similar type based on a common schema referred to as ontology. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
+        "operationId": "Features_Get",
+        "x-ms-examples": {
+          "GetFeature": {
+            "$ref": "./examples/features/Features_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/FeatureId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FeatureResult"
```

Comment:
Should we document what exact links should be in Feature Result. 
Eg: GET Collections will always return the specific 3 links. Similarly, GET Feature would also return specific links. Maybe documenting them would help users determine how to structure their code, rather than them making calls and figuring out what links are returned when and what to do with them.

Reply:
As discussed offline, I've updated the description of the FeaturesEndpointLink definition to reference the OGC spec where there's more details about the link relations.